analyses/cluster-cell-calling/README.md

```diff
@@ -0,0 +1,66 @@
+
+To run all of the Rscripts in this module sequentially on an interactive session on HPC, please run the following command from an interactive compute node:
+
+```
+bash run-cluster-cell-calling.sh
+```
+
+### Run module by using lsf on HPC within the container
+
+There is also the option to run a lsf job on the HPC cluster by using the following command on an HPC node:
+
+```
+bsub < lsf-script.txt
+```
+
+## Folder content
+This folder contains a script tasked to calculate clusters and find markers for each cluster across the project.
+
+## Analysis strategy:
+We recommend the user to follow the following steps for running the current module:
+- Step (1) `run-cluster-cell-calling-step1.R`: At first, the `01-cluster-cell-calling.Rmd` should be run for a set of resolutions or by default.
+- Step (2) `run-cluster-cell-calling-step2.R`: After inspection of the first round of results, the single resolution that fits best the data can be provided and used to run the `02-find-markers.Rmd`.
+
+In order to read the clustertree and decide properly on the resolution, user should consider tissue type, total number of cells in the dataset, and expected known cell types in there (i.e., tissue ecosystem). That would help to determine the correct number of clusters in a biologically meaningful way, considering known cell types and avoiding splitting clusters into unstable, small ones. That can also help to explore smaller clusters that might contain unknown or disease/patient-specific clusters that are still worth considering and investigating from the research/clinical perspective. 
+
+After inspection of the two rounds of results in the `cluster-cell-calling` module, user can remove clusters if necessary. This is relevant to projects in which we need to remove clusters from the object. This is the case, e.g., in PDX projects, there might be both human and mouse clusters identified after the `02-find-markers.Rmd` step of the the `cluster-cell-calling` module. In this case, we recommend the user to run the `contamination-remove-cells-analysis` module that allows to remove clusters, repeat normalization and integration steps. This object can then be used for cell type annotation or other type of analysis.
+
+
+## Folder structure 
+
+The structure of this folder is as follows:
+
+```
+├── 01-cluster-cell-calling.Rmd
+├── 02-find-markers.Rmd
+├── lsf-script.txt
+├── plots
+|   ├── 01_cluster_cell_calling_{resolution}
+|   ├── 02_find_markers
+|   ├── Report_cluster_cell_calling_{resolution}_<Sys.Date()>.html
+|   ├── Report_cluster_cell_calling_{resolution}_<Sys.Date()>.pdf
+|   ├── Report_find_markers_<Sys.Date()>.html
+|   └── Report_find_markers_<Sys.Date()>.pdf
+├── README.md
+├── results
+|   ├── 01_cluster_cell_calling_{resolution}
+|   └── 02_find_markers
+├── run-cluster-cell-calling-step1.R
+├── run-cluster-cell-calling-step2.R
+├── run-cluster-cell-calling-step1.sh
+├── run-cluster-cell-calling-step2.sh
```

Comment:
I think the file structure needs to be updated to reflect that there is only the `run-cluster-cell-calling.sh` in the module, not separate `run-cluster-cell-calling-step1.sh` and `run-cluster-cell-calling-step2.sh` files.

Reply:
```suggestion
```